### ISSUE TITLE
exec: Unable to find a path to the node binary. Please manually set config.execPath

### ISSUE THREAD

**Issue by kshestakov** [GoogleCloudPlatform/cloud-code-vscode#169]
**Environment**:
Extension version: 1.0.1
VSCode version: 1.40.2
OS: darwin (MacOS)

**Description**:
Such an error is poped up randomly while I switch to another .yaml file (e.g. file with deployment description):
```exec: Unable to find a path to the node binary. Please manually set config.execPath```

**Repro step**:
Simply switch to a new tab with yaml file opened

**Comment by redborian** [comment]
Thanks for reporting the issue @kshestakov. Is this error blocking your workflow in any way? How often do you get this issue? Can you set `cloudcode.yaml.trace.server` setting to ` verbose` and copy paste the logs from "Cloud Code Yaml Support" output stream?

**Comment by kshestakov** [comment]
>Is this error blocking your workflow in any way?
It constantly opens bottom panel that is getting annoying
>How often do you get this issue?
Unfortunately, pretty often to become annoying :(
>Can you set cloudcode.yaml.trace.server setting to verbose and copy paste the logs from "Cloud Code Yaml Support" output stream?
Sure.
Output:
```[Trace - 2:57:26 PM] Sending notification 'textDocument/didOpen'.
Params: {
    "textDocument": {
        "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
        "languageId": "yaml",
        "version": 1,
        "text": "apiVersion: apps/v1\nkind: Deployment\nmetadata:\n  name: deployment\n  labels:\n    app: aproxy\nspec:\n  minReadySeconds: 60 # Number of seconds to wait after a Pod is created and its status is Ready\n  replicas: 3\n  selector:\n    matchLabels:\n      app: aproxy\n  strategy:\n    type: RollingUpdate\n    rollingUpdate:\n      maxSurge: 100%\n      maxUnavailable: 0\n  template:\n    metadata:\n      labels:\n        app: aproxy\n    spec:\n      containers:\n        - name: container\n          image: gcr.io/apomaya/aproxy:v2.3.2-0-g26904f2\n          resources:\n            requests:\n              memory: \"0.375G\"\n              cpu: \"0.5\"\n          ports:\n            - containerPort: 80\n          volumeMounts:\n            - name: config-volume\n              mountPath: /aproxy/v2/config/prod.json\n              subPath: prod.json\n          readinessProbe:\n            httpGet:\n              path: /aproxy-status.json\n              port: 80\n            initialDelaySeconds: 3\n            periodSeconds: 30 # defaults to 10\n            timeoutSeconds: 3 # defaults to 1\n          livenessProbe:\n            httpGet:\n              path: /aproxy-status.json\n              port: 80\n            initialDelaySeconds: 60\n            periodSeconds: 30 # defaults to 10\n            timeoutSeconds: 3 # defaults to 1\n      terminationGracePeriodSeconds: 8 # Number of seconds to wait for connections to terminate before shutting down Pods\n      volumes:\n        - name: config-volume\n          configMap:\n            name: config\n"
    }
}


[Trace - 2:57:26 PM] Sending request 'textDocument/documentLink - (6541)'.
Params: {
    "textDocument": {
        "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml"
    }
}


[Trace - 2:57:26 PM] Sending request 'textDocument/documentSymbol - (6542)'.
Params: {
    "textDocument": {
        "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml"
    }
}


[Trace - 2:57:26 PM] Received notification 'telemetry/event'.
Params: {
    "duration_ms": "0",
    "event_name": "cloudcode.lsp.documentlink.created"
}


[Trace - 2:57:26 PM] Received response 'textDocument/documentLink - (6541)' in 55ms.
Result: [
    {
        "range": {
            "start": {
                "line": 32,
                "character": 20
            },
            "end": {
                "line": 32,
                "character": 33
            }
        }
    },
    {
        "range": {
            "start": {
                "line": 51,
                "character": 16
            },
            "end": {
                "line": 51,
                "character": 29
            }
        }
    },
    {
        "range": {
            "start": {
                "line": 53,
                "character": 18
            },
            "end": {
                "line": 53,
                "character": 24
            }
        }
    }
]


[Trace - 2:57:26 PM] Received notification 'telemetry/event'.
Params: {
    "duration_ms": "1",
    "event_name": "cloudcode.lsp.documentsymbol.created"
}


[Trace - 2:57:26 PM] Received response 'textDocument/documentSymbol - (6542)' in 75ms.
Result: [
    {
        "name": "apiVersion",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 0,
                    "character": 0
                },
                "end": {
                    "line": 0,
                    "character": 19
                }
            }
        }
    },
    {
        "name": "kind",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 1,
                    "character": 0
                },
                "end": {
                    "line": 1,
                    "character": 16
                }
            }
        }
    },
    {
        "name": "metadata",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 2,
                    "character": 0
                },
                "end": {
                    "line": 5,
                    "character": 15
                }
            }
        }
    },
    {
        "name": "name",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 3,
                    "character": 2
                },
                "end": {
                    "line": 3,
                    "character": 18
                }
            }
        },
        "containerName": "metadata"
    },
    {
        "name": "labels",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 4,
                    "character": 2
                },
                "end": {
                    "line": 5,
                    "character": 15
                }
            }
        },
        "containerName": "metadata"
    },
    {
        "name": "app",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 5,
                    "character": 4
                },
                "end": {
                    "line": 5,
                    "character": 15
                }
            }
        },
        "containerName": "metadata.labels"
    },
    {
        "name": "spec",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 6,
                    "character": 0
                },
                "end": {
                    "line": 54,
                    "character": 0
                }
            }
        }
    },
    {
        "name": "minReadySeconds",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 7,
                    "character": 2
                },
                "end": {
                    "line": 7,
                    "character": 21
                }
            }
        },
        "containerName": "spec"
    },
    {
        "name": "replicas",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 8,
                    "character": 2
                },
                "end": {
                    "line": 8,
                    "character": 13
                }
            }
        },
        "containerName": "spec"
    },
    {
        "name": "selector",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 9,
                    "character": 2
                },
                "end": {
                    "line": 11,
                    "character": 17
                }
            }
        },
        "containerName": "spec"
    },
    {
        "name": "matchLabels",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 10,
                    "character": 4
                },
                "end": {
                    "line": 11,
                    "character": 17
                }
            }
        },
        "containerName": "spec.selector"
    },
    {
        "name": "app",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 11,
                    "character": 6
                },
                "end": {
                    "line": 11,
                    "character": 17
                }
            }
        },
        "containerName": "spec.selector.matchLabels"
    },
    {
        "name": "strategy",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 12,
                    "character": 2
                },
                "end": {
                    "line": 16,
                    "character": 23
                }
            }
        },
        "containerName": "spec"
    },
    {
        "name": "type",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 13,
                    "character": 4
                },
                "end": {
                    "line": 13,
                    "character": 23
                }
            }
        },
        "containerName": "spec.strategy"
    },
    {
        "name": "rollingUpdate",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 14,
                    "character": 4
                },
                "end": {
                    "line": 16,
                    "character": 23
                }
            }
        },
        "containerName": "spec.strategy"
    },
    {
        "name": "maxSurge",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 15,
                    "character": 6
                },
                "end": {
                    "line": 15,
                    "character": 20
                }
            }
        },
        "containerName": "spec.strategy.rollingUpdate"
    },
    {
        "name": "maxUnavailable",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 16,
                    "character": 6
                },
                "end": {
                    "line": 16,
                    "character": 23
                }
            }
        },
        "containerName": "spec.strategy.rollingUpdate"
    },
    {
        "name": "template",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 17,
                    "character": 2
                },
                "end": {
                    "line": 54,
                    "character": 0
                }
            }
        },
        "containerName": "spec"
    },
    {
        "name": "metadata",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 18,
                    "character": 4
                },
                "end": {
                    "line": 20,
                    "character": 19
                }
            }
        },
        "containerName": "spec.template"
    },
    {
        "name": "labels",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 19,
                    "character": 6
                },
                "end": {
                    "line": 20,
                    "character": 19
                }
            }
        },
        "containerName": "spec.template.metadata"
    },
    {
        "name": "app",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 20,
                    "character": 8
                },
                "end": {
                    "line": 20,
                    "character": 19
                }
            }
        },
        "containerName": "spec.template.metadata.labels"
    },
    {
        "name": "spec",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 21,
                    "character": 4
                },
                "end": {
                    "line": 54,
                    "character": 0
                }
            }
        },
        "containerName": "spec.template"
    },
    {
        "name": "containers",
        "kind": 18,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 22,
                    "character": 6
                },
                "end": {
                    "line": 49,
                    "character": 6
                }
            }
        },
        "containerName": "spec.template.spec"
    },
    {
        "name": "name",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 23,
                    "character": 10
                },
                "end": {
                    "line": 23,
                    "character": 25
                }
            }
        },
        "containerName": "spec.template.spec.containers"
    },
    {
        "name": "image",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 24,
                    "character": 10
                },
                "end": {
                    "line": 24,
                    "character": 56
                }
            }
        },
        "containerName": "spec.template.spec.containers"
    },
    {
        "name": "resources",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 25,
                    "character": 10
                },
                "end": {
                    "line": 28,
                    "character": 24
                }
            }
        },
        "containerName": "spec.template.spec.containers"
    },
    {
        "name": "requests",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 26,
                    "character": 12
                },
                "end": {
                    "line": 28,
                    "character": 24
                }
            }
        },
        "containerName": "spec.template.spec.containers.resources"
    },
    {
        "name": "memory",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 27,
                    "character": 14
                },
                "end": {
                    "line": 27,
                    "character": 30
                }
            }
        },
        "containerName": "spec.template.spec.containers.resources.requests"
    },
    {
        "name": "cpu",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 28,
                    "character": 14
                },
                "end": {
                    "line": 28,
                    "character": 24
                }
            }
        },
        "containerName": "spec.template.spec.containers.resources.requests"
    },
    {
        "name": "ports",
        "kind": 18,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 29,
                    "character": 10
                },
                "end": {
                    "line": 31,
                    "character": 10
                }
            }
        },
        "containerName": "spec.template.spec.containers"
    },
    {
        "name": "containerPort",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 30,
                    "character": 14
                },
                "end": {
                    "line": 30,
                    "character": 31
                }
            }
        },
        "containerName": "spec.template.spec.containers.ports"
    },
    {
        "name": "volumeMounts",
        "kind": 18,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 31,
                    "character": 10
                },
                "end": {
                    "line": 35,
                    "character": 10
                }
            }
        },
        "containerName": "spec.template.spec.containers"
    },
    {
        "name": "name",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 32,
                    "character": 14
                },
                "end": {
                    "line": 32,
                    "character": 33
                }
            }
        },
        "containerName": "spec.template.spec.containers.volumeMounts"
    },
    {
        "name": "mountPath",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 33,
                    "character": 14
                },
                "end": {
                    "line": 33,
                    "character": 52
                }
            }
        },
        "containerName": "spec.template.spec.containers.volumeMounts"
    },
    {
        "name": "subPath",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 34,
                    "character": 14
                },
                "end": {
                    "line": 34,
                    "character": 32
                }
            }
        },
        "containerName": "spec.template.spec.containers.volumeMounts"
    },
    {
        "name": "readinessProbe",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 35,
                    "character": 10
                },
                "end": {
                    "line": 41,
                    "character": 29
                }
            }
        },
        "containerName": "spec.template.spec.containers"
    },
    {
        "name": "httpGet",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 36,
                    "character": 12
                },
                "end": {
                    "line": 38,
                    "character": 22
                }
            }
        },
        "containerName": "spec.template.spec.containers.readinessProbe"
    },
    {
        "name": "path",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 37,
                    "character": 14
                },
                "end": {
                    "line": 37,
                    "character": 39
                }
            }
        },
        "containerName": "spec.template.spec.containers.readinessProbe.httpGet"
    },
    {
        "name": "port",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 38,
                    "character": 14
                },
                "end": {
                    "line": 38,
                    "character": 22
                }
            }
        },
        "containerName": "spec.template.spec.containers.readinessProbe.httpGet"
    },
    {
        "name": "initialDelaySeconds",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 39,
                    "character": 12
                },
                "end": {
                    "line": 39,
                    "character": 34
                }
            }
        },
        "containerName": "spec.template.spec.containers.readinessProbe"
    },
    {
        "name": "periodSeconds",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 40,
                    "character": 12
                },
                "end": {
                    "line": 40,
                    "character": 29
                }
            }
        },
        "containerName": "spec.template.spec.containers.readinessProbe"
    },
    {
        "name": "timeoutSeconds",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 41,
                    "character": 12
                },
                "end": {
                    "line": 41,
                    "character": 29
                }
            }
        },
        "containerName": "spec.template.spec.containers.readinessProbe"
    },
    {
        "name": "livenessProbe",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 42,
                    "character": 10
                },
                "end": {
                    "line": 48,
                    "character": 29
                }
            }
        },
        "containerName": "spec.template.spec.containers"
    },
    {
        "name": "httpGet",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 43,
                    "character": 12
                },
                "end": {
                    "line": 45,
                    "character": 22
                }
            }
        },
        "containerName": "spec.template.spec.containers.livenessProbe"
    },
    {
        "name": "path",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 44,
                    "character": 14
                },
                "end": {
                    "line": 44,
                    "character": 39
                }
            }
        },
        "containerName": "spec.template.spec.containers.livenessProbe.httpGet"
    },
    {
        "name": "port",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 45,
                    "character": 14
                },
                "end": {
                    "line": 45,
                    "character": 22
                }
            }
        },
        "containerName": "spec.template.spec.containers.livenessProbe.httpGet"
    },
    {
        "name": "initialDelaySeconds",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 46,
                    "character": 12
                },
                "end": {
                    "line": 46,
                    "character": 35
                }
            }
        },
        "containerName": "spec.template.spec.containers.livenessProbe"
    },
    {
        "name": "periodSeconds",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 47,
                    "character": 12
                },
                "end": {
                    "line": 47,
                    "character": 29
                }
            }
        },
        "containerName": "spec.template.spec.containers.livenessProbe"
    },
    {
        "name": "timeoutSeconds",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 48,
                    "character": 12
                },
                "end": {
                    "line": 48,
                    "character": 29
                }
            }
        },
        "containerName": "spec.template.spec.containers.livenessProbe"
    },
    {
        "name": "terminationGracePeriodSeconds",
        "kind": 16,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 49,
                    "character": 6
                },
                "end": {
                    "line": 49,
                    "character": 38
                }
            }
        },
        "containerName": "spec.template.spec"
    },
    {
        "name": "volumes",
        "kind": 18,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 50,
                    "character": 6
                },
                "end": {
                    "line": 54,
                    "character": 0
                }
            }
        },
        "containerName": "spec.template.spec"
    },
    {
        "name": "name",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 51,
                    "character": 10
                },
                "end": {
                    "line": 51,
                    "character": 29
                }
            }
        },
        "containerName": "spec.template.spec.volumes"
    },
    {
        "name": "configMap",
        "kind": 2,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 52,
                    "character": 10
                },
                "end": {
                    "line": 53,
                    "character": 24
                }
            }
        },
        "containerName": "spec.template.spec.volumes"
    },
    {
        "name": "name",
        "kind": 15,
        "location": {
            "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
            "range": {
                "start": {
                    "line": 53,
                    "character": 12
                },
                "end": {
                    "line": 53,
                    "character": 24
                }
            }
        },
        "containerName": "spec.template.spec.volumes.configMap"
    }
]


[Trace - 2:57:27 PM] Sending request 'textDocument/codeLens - (6543)'.
Params: {
    "textDocument": {
        "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml"
    }
}


[Trace - 2:57:27 PM] Received notification 'textDocument/publishDiagnostics'.
Params: {
    "uri": "file:///Users/Kirill/Projects/Apomaya/apomaya.clusters/gcloud/apomaya-staging/staging-cluster/deployment.yaml",
    "diagnostics": []
}


[Trace - 2:57:27 PM] Received notification 'window/logMessage'.
Params: {
    "type": 1,
    "message": "exec: Unable to find a path to the node binary. Please manually set config.execPath"
}


[Error - 2:57:27 PM] exec: Unable to find a path to the node binary. Please manually set config.execPath
[Trace - 2:57:27 PM] Received notification 'telemetry/event'.
Params: {
    "duration_ms": "3",
    "event_name": "cloudcode.lsp.codelens.created",
    "failure_reason": "unknown"
}


[Trace - 2:57:27 PM] Received response 'textDocument/codeLens - (6543)' in 19ms.
Result: []
```

**Comment by redborian** [comment]
Thanks for the logs @kshestakov. We will fix this in the upcoming release

**Comment by redborian** [comment]
This issue seems to be due to one of our dependent packages leveraging shelljs. Our next release will stop focusing YAML logs if any error appears. That should slightly alleviate this problem. It will give us more time to come up with a good fix for this issue.

**Comment by jmah-cc** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-vscode-internal](https://github.com/GoogleCloudPlatform/cloud-code-vscode-internal/issues/1845)

**Comment by jmann-ce** [comment]
Curious on not setting focus on this... 

I'm still getting this issue on this version of vs-code:
Version: 1.42.0
Commit: ae08d5460b5a45169385ff3fd44208f431992451
Date: 2020-02-06T10:51:33.119Z
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.3.0

**Comment by jeremychone** [comment]
Same here
Version: 1.42.0
Commit: ae08d5460b5a45169385ff3fd44208f431992451
Date: 2020-02-06T10:51:33.119Z
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Darwin x64 19.3.0

**Comment by redborian** [comment]
This should be fixed in the latest version of Cloud Code (v1.2.0). Feel free to reopen this issue if you still see the issue.
